src/main/java/com/hbm/items/tool/ItemBombCaller.java

```diff
@@ -16,101 +17,81 @@
 import net.minecraft.world.World;
 
 public class ItemBombCaller extends Item {
-	
+
 	public ItemBombCaller() {
 		super();
-        this.setHasSubtypes(true);
+		this.setHasSubtypes(true);
 	}
-	
+
 	@Override
-	public void addInformation(ItemStack itemstack, EntityPlayer player, List list, boolean bool)
+	public void addInformation(ItemStack stack, EntityPlayer player, List list, boolean bool)
 	{
 		list.add("Aim & click to call an airstrike!");
 
-        if(itemstack.getItemDamage() == 0)
-    		list.add("Type: Carpet bombing");
-        if(itemstack.getItemDamage() == 1)
-    		list.add("Type: Napalm");
-        if(itemstack.getItemDamage() == 2)
-    		list.add("Type: Poison gas");
-        if(itemstack.getItemDamage() == 3)
-    		list.add("Type: Agent orange");
-        if(itemstack.getItemDamage() == 4)
-    		list.add("Type: Atomic bomb");
-        if(itemstack.getItemDamage() == 5)
-    		list.add("Type: VT stinger rockets");
-        if(itemstack.getItemDamage() == 6)
-    		list.add("Type: PIP OH GOD");
-        if(itemstack.getItemDamage() == 7)
-    		list.add("Type: Cloud the cloud oh god the cloud");
+		switch (stack.getItemDamage()) {
+			case 0: list.add("Type: Carpet bombing"); break;
+			case 1: list.add("Type: Napalm"); break;
+			case 2: list.add("Type: Poison gas"); break;
+			case 3: list.add("Type: Agent orange"); break;
+			case 4: list.add("Type: Atomic bomb"); break;
+			case 5: list.add("Type: VT stinger rockets"); break;
+			case 6: list.add("Type: PIP OH GOD"); break;
+			case 7: list.add("Type: Cloud the cloud oh god the cloud"); break;
+			default: list.add("Type: INVALID, Report it to mod creator");
```

Comment:
Weird default case, I don't need people to report to me when they /give themselves airstrikes with weird meta.

Reply:
Some failsafe didn't seem like a bad idea in retrospect, incase some airstrike is added